README.md

```diff
@@ -255,84 +264,160 @@ See below for example json.
 
 --
 
-### Add subscribers to a push queue
+### Update a Queue
 
 ```ruby
-subscribers = [
-    {
-        name: 'first',
-        url: 'http://first.endpoint.xx/process',
-        headers: {
-            Content-Type: 'application/json'
-        }
-    },
-    {
-        name: 'second',
-        url: 'http://second.endpoint.xx/process',
-    }
-]
-queue.add_subscribers(subscribers)
+queue = ironmq.update_queue('my-queue', message_timeout: 300) # => #<IronMQ::Queue:...>
+# or
+queue.update!(message_timeout: 600) # => {"queue" => {"name": "my-queue", ...}}
```

Comment:
I do not think this is good idea. If request fails, local object will contain wrong info about the queue. Current implementation relies on API response.

I made `Queue` and `Message` mostly immutable from the user perspective. Update the instance by bang methods or make new object of appropriate class. It can save customers a lot of time of debug their own apps (particularly, concurrent).